dipy/viz/colormap.py

```diff
@@ -1,27 +1,30 @@
+from __future__ import division
+
 import numpy as np
 
 
-def cc(na,nd):
-    return ( na * np.cos( nd * np.pi/180.0 ) );
+def cc(na, nd):
+    return na * np.cos(nd * np.pi/180.0)
 
-def ss(na,nd):
-    return na * np.sin( nd * np.pi/180.0 ) ;
 
-def boys2rgb(v):
+def ss(na, nd):
+    return na * np.sin(nd * np.pi/180.0)
 
+
+def boys2rgb(v):
     """ boys 2 rgb cool colormap
-    
-    Maps a given field of undirected lines (line field) to rgb 
-    colors using Boy's Surface immersion of the real projective 
-    plane. 
-    Boy's Surface is one of the three possible surfaces 
-    obtained by gluing a Mobius strip to the edge of a disk. 
-    The other two are the crosscap and Roman surface, 
-    Steiner surfaces that are homeomorphic to the real 
-    projective plane (Pinkall 1986). The Boy's surface 
-    is the only 3D immersion of the projective plane without 
-    singularities. 
-    Visit http://www.cs.brown.edu/~cad/rp2coloring for further details. 
+
+    Maps a given field of undirected lines (line field) to rgb
+    colors using Boy's Surface immersion of the real projective
+    plane.
+    Boy's Surface is one of the three possible surfaces
+    obtained by gluing a Mobius strip to the edge of a disk.
+    The other two are the crosscap and Roman surface,
+    Steiner surfaces that are homeomorphic to the real
+    projective plane (Pinkall 1986). The Boy's surface
+    is the only 3D immersion of the projective plane without
+    singularities.
+    Visit http://www.cs.brown.edu/~cad/rp2coloring for further details.
```

Comment:
Can you please change the address with this one? http://hci.stanford.edu/~cagatay//rp2coloring/index.html